index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Jordan Addis - Portfolio</title>
+	<!-- Styling -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<!-- Fonts -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+	<!-- Scripts -->
+	<script src="https://kit.fontawesome.com/7c6af540e7.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<a class="anchor" id="top"></a>
+	<nav>
+		<div class="navLeft">
+			<a href="#top" class="navLogo">
```

Comment:
I would say that for UI & UX it isn't obvious you should click your name to enact anything; whilst it is a clean and convenient way to do it without adding a fixed button, maybe it would be worth considering a hover effect so that it is apparent something happens as a result of pressing.